tutorindigo/templates/indigo/lms/templates/instructor/instructor_dashboard_2/course_info.html

```diff
@@ -0,0 +1,170 @@
+<%page args="section_data" expression_filter="h"/>
```

Comment:
What part(s) of this file where modified, compared to upstream? As usual, we should hightlight them with a comment. I couldn't find such a comment, but maybe I didn't look straight ;)

Reply:
Regis, there is no comment. We wrapped a heading (h4) and `.enrollment-wrapper` div in `course-info` div for styling. I have removed this file as it doesn't seem good to override the file for adding a new div. Thanks for pointing it out.